src/main/java/mate/academy/spring/service/impl/UserServiceImpl.java

```diff
@@ -1,14 +1,17 @@
-package mate.academy.spring.service;
+package mate.academy.spring.service.impl;
 
 import java.util.List;
 import mate.academy.spring.dao.UserDao;
 import mate.academy.spring.model.User;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service
 public class UserServiceImpl implements UserService {
     private final UserDao userDao;
 
+    @Autowired
```

Comment:
if only 1 constructor do we need @Autowired?

Reply:
Yes, it not nessesary, if we had 1 constructor, but we can use it) if we use annotation @Autowired in one class, we use it everywhere.